packages/platforms/inventec/x86-64/x86-64-inventec-d7032q28b/onlp/builds/src/module/src/sfpi.c

```diff
@@ -1,26 +1,11 @@
 /************************************************************
- * <bsn.cl fy=2014 v=onl>
+ * sfpi.c
```

Comment:
It appears all SFP support has been removed as part of this pull request. Is this intentional, and if so why?